### PR TITLE
Add Quote On Logout & Fix Mood Rating Logic

### LINKED ISSUE #2: User can Logout
As a user, I can be on any page, I want to click "logout" so I can exit the app

- [x] create navbar with logout link
- [ ] redirect user to welcome page and display goodbye quote

### PR DESCRIPTION
**Pull request by javier-aguilar**
## What functionality does this accomplish?
- Add Random Quote 
- Update logic on mood rating
- Update application timezone
closes #2 
**Description:**
Display quote when user logs out

### Test Coverage Percentage: 100.0%
- [ ] No tests have been changed
- [x] Some tests have been changed (Update logout spec)
- [ ] All of the tests have been changed(Please describe what in the world happened):
### Rubocop:
- [x] No offenses
- [ ] Offenses (number of offenses: _ )
## Checklist:
- [ ] My code has no unused/commented out code
- [x] I have reviewed my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have fully tested my code
- [ ] I have partially tested my code (please explain why):

### CODE REVIEW

File: spec/features/user/logout_spec.rb [hunk 1]
```diff
@@ -1,19 +1,21 @@
...
     stub_omniauth
-    
+
     visit '/login'
 
     click_on 'Sign In With Google'
 
     click_link 'Logout'
-    
-    expect(current_path).to eq(root_path)
```
Comment (javier-aguilar) [inline spec/features/user/logout_spec.rb:13]: I had to remove this since I'm using render instead of re-direct which caused the test to fail.

File: app/models/user.rb [hunk 2]
```diff
@@ -40,9 +40,9 @@ def suggested_videos
     rating = mood_rating
     if rating < 1.5
       VideoResults.new.get_suggested_videos('sad')
-    elsif rating > 1.5 && rating < 2.25
+    elsif rating >= 1.5 && rating < 2.25
```
Comment (danielbldr) [inline app/models/user.rb:43]: Good catch on this - glad I had a rating of 1.5 during our demo to catch this edge case

File: config/application.rb [hunk 3]
```diff
@@ -27,5 +27,8 @@ class Application < Rails::Application
 
     # Don't generate system test files.
     config.generators.system_tests = nil
+
+    config.time_zone = 'Mountain Time (US & Canada)'
+
   end
```
Comment (SMJ289) [inline config/application.rb:33]: What does this do?

File: spec/features/user/login_spec.rb [hunk 4]
```diff
@@ -9,9 +9,9 @@
     expect(current_path).to eq('/login')
   end
 
-  it 'when I visit the login page I can login with Google and the Sign In button now says Logout' do
+  it 'when I visit the login page I can login with Google and the Sign In button now says Logout', :vcr do
```
Comment (SMJ289) [inline spec/features/user/login_spec.rb:12]: What purpose does VCR serve in this test that the omniauth stub doesn't cover?
